SConstruct

```diff
@@ -79,4 +86,4 @@ def generate_qtbuilddir():
         os.symlink(prefix, standartprefix)
 
 if os.name == 'posix' and qt_version and not 'CONDA_BUILD' in os.environ:
-    generate_qtbuilddir()
\ No newline at end of file
+    generate_qtbuilddir()
```

Comment:
What does that mean? We are trying to build PlantGL. So why do we need to generate a qt build dir?

Reply:
Pls, add doc to the function